depsolver.js

```diff
@@ -193,7 +193,7 @@ module.exports = function depSolver (executable, libs, parallel, cb) {
              * sign those anyways, just ensure to
              * sign them before the app executable
              */
-            finalLibs.unshift(...orphaned);
+            finalLibs.unshift(orphaned);
```

Comment:
This is not correct. doesn't have the same meaning. If your nodejs doesnt supports ES6 syntax, please update :)

Reply:
Thanks. Version details have been provided in issue #70.